packages/suite/src/components/suite/AccountStickyContent/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FiatValue, FormattedCryptoAmount, AccountLabeling, Translation } from '@suite-components';
+import { CoinLogo, variables } from '@trezor/components';
+import { Account } from '@wallet-types';
+
+const Main = styled.div`
+    align-items: center;
+    display: flex;
+`;
+
+const BalanceWrapperContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin: 0 0 0 13px;
+`;
+
+const Balance = styled.div`
+    white-space: nowrap;
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+const FiatBalanceWrapper = styled.div`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    margin-left: 0.5ch;
+`;
+
+const LabelWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin: 0 13px 0 0;
+`;
+
+const BalanceInner = styled.div`
+    display: flex;
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const StyledFiatValue = styled.div`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const StyledFormTitle = styled.h3`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+const StyledSeparator = styled.span`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    font-size: 31px;
+    font-family: initial;
+    display: inline-block;
+    line-height: 1;
+    padding: 0 8px;
+`;
+
+interface Props {
+    account?: Account;
+    routeName: string;
```

Comment:
It would be safer to have this more strictly typed as eg.
```suggestion
    routeName: Route['name'];
```

where `Route` is imported from `constants/suite/route.ts`.